.github/release-please-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "packages": {
```

Comment:
I think you might need to provide a `bootstrap-sha` to a recent commit, so that Release Please doesn't try to parse the entire git history to determine the next version.

example here:
https://github.com/krystal/checken/blob/master/.github/release-please-config.json#L2

Then when you want to release the version you can do it manually with an empty commit:

https://github.com/googleapis/release-please?tab=readme-ov-file#how-do-i-change-the-version-number

> git commit --allow-empty -m "chore: release 1.2.3" -m "Release-As: 1.2.3"

Obviously choose whatever version number you think is correct.